bin/role2docker

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
```

Comment:
It's a tricky script, so add a global description including the big steps

Reply:
I have added a link to the doc.